client/src/snippets.json

```diff
@@ -2,40 +2,58 @@
     "Basic Galaxy tool": {
         "prefix": "gx-tool",
         "body": [
-            "<tool id=\"$1\" name=\"$2\" version=\"@TOOL_VERSION@+galaxy@VERSION_SUFFIX@\" profile=\"20.01\" license=\"${4|MIT,Apache-2.0,GPL-3.0-or-later,GPL-2.0-only,BSD-3-Clause,LGPL-3.0+|}\">",
+            "<tool id=\"$1\" name=\"$2\" version=\"@TOOL_VERSION@+galaxy@VERSION_SUFFIX@\" profile=\"20.01\" license=\"${3|MIT,Apache-2.0,GPL-3.0-or-later,GPL-2.0-only,BSD-3-Clause,LGPL-3.0+|}\">",
             "    <description>$5</description>",
             "    <xrefs>",
             "        <xref type=\"bio.tools\">$0</xref>",
             "    </xrefs>",
             "    <macros>",
-            "        <token name=\"@TOOL_VERSION@\">${3:0.1.0}</token>",
+            "        <token name=\"@TOOL_VERSION@\">${4:0.1.0}</token>",
             "        <token name=\"@VERSION_SUFFIX@\">0</token>",
             "    </macros>",
             "    <!-- TODO: please annotate this tool with topics and operations from http://edamontology.org -->",
             "    <!-- TODO: for more information see: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#edam-topics-and-operations -->",
             "    <edam_topics>",
-            "        <edam_topic></edam_topic>",
+            "        <edam_topic>topic_TODO</edam_topic>",
             "    </edam_topics>",
             "    <edam_operations>",
-            "        <edam_operation></edam_operation>",
+            "        <edam_operation>operation_TODO</edam_operation>",
             "    </edam_operations>",
             "    <requirements>",
             "        <requirement type=\"package\" version=\"@TOOL_VERSION@\">$1</requirement>",
             "    </requirements>",
             "    <command detect_errors=\"exit_code\"><![CDATA[",
-            "        TODO: Fill in command template.",
+            "        ## TODO: Fill in command using Cheetah templates",
+            "        ## Hint: Use [ctrl+alt+c] after defining the inputs/outputs to auto-generate some Cheetah boilerplate code for you.",
             "    ]]></command>",
             "    <inputs>",
             "    </inputs>",
             "    <outputs>",
             "    </outputs>",
             "    <tests>",
+            "        <!-- Hint: You can use [ctrl+alt+t] after defining the inputs/outputs to auto-scaffold some basic test cases. -->",
             "    </tests>",
             "    <help><![CDATA[",
-            "        TODO: Fill in help in reStructuredText (https://en.wikipedia.org/wiki/ReStructuredText)",
+            "",
+            ".. class:: infomark",
+            "",
+            "**What it does**",
+            "",
+            "TODO: Fill in help in reStructuredText format (https://en.wikipedia.org/wiki/ReStructuredText)",
```

Comment:
Do we want to advertise http://rst.ninjs.org until GLS supports RST visualization?

Reply:
Sure, why not? :)